.github/workflows/lint-checker.yml

```diff
@@ -0,0 +1,28 @@
+name: lint-checker-ci
+
+on:
+  push:
+    branches:
+      - master
+      - chore/gofmt
```

Comment:
we should remove this line?

Reply:
this is for verifying this workflow is in use.

Reply:
Will remove it after merging.